Taskfile.yml

```diff
@@ -0,0 +1,24 @@
+version: '3'
+
+tasks:
+  check:style:
+    desc: Check Conding style
+    cmds:
+      - poetry run pylama
+
+  install:
+    desc: Install all poetry dependencies
+    cmds:
+      - poetry install
+
+  build:
+    desc: Build the whl package
+    cmds:
+      - poetry build
+
+  build:docker:
+    desc: Build the docker image
+    deps:
+      - task: build
+    cmds:
+      - docker build --build-context dist=./dist -t aksw/quitstore .
```

Comment:
Throws an error "unknown flag: --build-context" but package and whl were built in "./dist"

Reply:
fixed